### ISSUE TITLE
toolbar buttons don't blur when they should

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#5631]
Clicking on a button in the toolbar will focus the button, and then clicking else where on the page/calendar should blur it. This is frequently not happening, probably for the same reason as #5628

1. Go to https://fullcalendar.io/docs/navLinks-demo
2. Click prev or next (should focus the button)
3. Click a navlink on the calendar

Result: button isn't blurred after click
Desired: button should be blurred (no longer focused)
